sdk/servicebus/service-bus/.prettierrc.json

```diff
@@ -1,6 +1,9 @@
 {
```

Comment:
Once we get this the rules worked out I think this should live at the scope of where all the source code that will share it lives, so I think that would put it at `sdk/.prettierrc.json`.

Reply:
I think the eslintrc.json file should likely be similarly scoped.

Reply:
Thank you @weshaggard